0.24.1: Tests fail: FindThreads only works if either C or CXX language is enabled

```
CMake Error at /usr/local/share/cmake/Modules/FindThreads.cmake:66 (message):
  FindThreads only works if either C or CXX language is enabled
Call Stack (most recent call first):
  CMakeLists.txt:13 (find_package)
```

clang-12
FreeBSD 13

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -9,9 +9,6 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-find_package(doctest            REQUIRED)
-find_package(Threads)
-
 if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     project(xtensor-test CXX)
 
@@ -20,6 +17,9 @@ if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     find_package(xtensor REQUIRED CONFIG)
     set(XTENSOR_INCLUDE_DIR ${xtensor_INCLUDE_DIRS})
 endif ()
+
+find_package(doctest            REQUIRED)
+find_package(Threads)
 
 if(NOT CMAKE_BUILD_TYPE)
     message(STATUS "Setting tests build type to Release")

